test/box-luatest/gh_10148_fix_crash_low_slab_alloc_factor_test.lua

```diff
@@ -0,0 +1,40 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.server = server:new({
+        box_cfg = {
```

Comment:
Let's set `wal_mode = 'none'` to speed up the test.

Reply:
Done.

```diff
@@ -8,6 +8,7 @@ g.before_all(function(cg)
         box_cfg = {
             slab_alloc_factor = 1.001,
             memtx_memory = 64 * 1024 * 1024,
+            wal_mode = 'none',
         }
     })
     cg.server:start()
```